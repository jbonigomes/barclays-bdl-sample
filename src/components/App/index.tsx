import React from 'react'

import Button from '../Button'
import QuickLink from '../QuickLink'
import TileSmall from '../TileSmall'
import TileDefault from '../TileDefault'
import TileWithBadge from '../TileWithBadge'
import TileWithGraphics from '../TileWithGraphics'

import styles from './index.module.scss'

import logo from '../../images/logo.png'
import graphic1 from '../../images/graphic_1.png'
import graphic2 from '../../images/graphic_2.png'
import graphic3 from '../../images/graphic_3.png'
import promoImage from '../../images/promo_image.png'

type AppProps = {
  onClick(e: React.MouseEvent<HTMLElement>): void,
}

const App: React.SFC<AppProps> = ({ onClick }: AppProps) => (
  <>
    <div className={styles.row}>
      <Button
        type="primary"
        text="CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        disabled
        type="primary"
        text="CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        type="secondary"
        text="CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        disabled
        type="secondary"
        text="CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        small
        type="primary"
        text="CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        small
        disabled
        type="primary"
        text="CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        small
        type="secondary"
        text="CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        small
        disabled
        type="secondary"
        text="CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        small
        type="primary"
        text="Small button"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        small
        text="Yes"
        type="primary"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        icon="card"
        type="primary"
        text="Button with icon"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        icon="share"
        type="secondary"
        text="Button with icon"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        disabled
        icon="transfer"
        type="primary"
        text="Button with icon"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        small
        icon="transfer"
        type="primary"
        text="Button with icon"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        type="primary"
        text="CTA CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <Button
        type="secondary"
        text="CTA CTA character limit 20-25"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <QuickLink
        icon="card"
        onClick={onClick}
        text="Report a debit card lost or stolen"
      />
    </div>

    <div className={styles.row}>
      <QuickLink
        icon="transfer"
        text="Make a transfer"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <QuickLink
        icon="share"
        text="Share"
        onClick={onClick}
      />
    </div>

    <div className={styles.row}>
      <TileWithGraphics
        image={graphic1}
        buttonText="View demo"
        title="See our tutorial"
        onClick={onClick}
      >
        Step by step illustrations on how to use Online Banking
      </TileWithGraphics>
    </div>

    <div className={styles.row}>
      <TileWithGraphics
        image={graphic2}
        title="Speed up login"
      >
        Save your details on the login screen and you'll only need your
        PINsentry device and card to login
      </TileWithGraphics>
    </div>

    <div className={styles.row}>
      <TileSmall
        image={graphic3}
        title="Overdraft"
        onClick={onClick}
      >
        Get an overdraft of up to <b>Â£2,000</b>
      </TileSmall>
    </div>

    <div className={styles.row}>
      <TileDefault
        image={promoImage}
        buttonText="Call to action"
        title="1st card title goes here"
        onTitleClick={onClick}
        onButtonClick={onClick}
      >
        Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse nec tortor
        urna. Ut laoreet sodales nisi, nex quis iaculi. Vestibulum rutrum quam
        vitae fringilla tincidunt. Suspendisse romani.
      </TileDefault>
    </div>

    <div className={styles.row}>
      <TileDefault
        image={promoImage}
        title="1st card title goes here"
        onTitleClick={onClick}
      >
        Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse nec tortor
        urna. Ut laoreet sodales nisi, nex quis iaculi. Vestibulum rutrum quam
        vitae fringilla tincidunt. Suspendisse romani.
      </TileDefault>
    </div>

    <div className={styles.row}>
      <TileWithBadge
        badge={logo}
        image={promoImage}
        title="Cashback"
        buttonText="Call to action"
        onTitleClick={onClick}
        onButtonClick={onClick}
      >
        Get cashback when shopping at participant retailers
      </TileWithBadge>
    </div>

    <div className={styles.row}>
      <TileWithBadge
        badge={logo}
        image={promoImage}
        title="Cashback"
        onTitleClick={onClick}
      >
        Get cashback when shopping at participant retailers
      </TileWithBadge>
    </div>
  </>
)

export default App
