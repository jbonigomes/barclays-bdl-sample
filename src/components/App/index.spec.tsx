import React from 'react'
import renderer from 'react-test-renderer'

import App from '.'

test('Match snapshot', () => {
  const props = {
    onClick: jest.fn(),
  }

  const tree = renderer.create(<App onClick={props.onClick}>Foo</App>)

  expect(tree.toJSON()).toMatchSnapshot()
})
