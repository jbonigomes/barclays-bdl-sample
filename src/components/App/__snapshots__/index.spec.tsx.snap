// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshot 1`] = `
Array [
  <div
    className="row"
  >
    <button
      className="primary button  "
      onClick={[MockFunction]}
    >
      CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="primary button  "
      disabled={true}
      onClick={[MockFunction]}
    >
      CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="secondary button  "
      onClick={[MockFunction]}
    >
      CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="secondary button  "
      disabled={true}
      onClick={[MockFunction]}
    >
      CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="primary button small "
      onClick={[MockFunction]}
    >
      CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="primary button small "
      disabled={true}
      onClick={[MockFunction]}
    >
      CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="secondary button small "
      onClick={[MockFunction]}
    >
      CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="secondary button small "
      disabled={true}
      onClick={[MockFunction]}
    >
      CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="primary button small "
      onClick={[MockFunction]}
    >
      Small button
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="primary button small "
      onClick={[MockFunction]}
    >
      Yes
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="primary button  "
      onClick={[MockFunction]}
    >
      <svg
        viewBox="0 0 21 14"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Card
        </title>
        <path
          d="M6.5,11H3c-0.3,0-0.5-0.2-0.5-0.5S2.7,10,3,10h3.5C6.8,10,7,10.2,7,10.5S6.8,11,6.5,11z"
        />
        <path
          d="M17.5,11h-7.1c-0.3,0-0.5-0.2-0.5-0.5s0.2-0.5,0.5-0.5h7.1c0.3,0,0.5,0.2,0.5,0.5S17.8,11,17.5,11z"
        />
        <path
          d="M6.5,7H3C2.7,7,2.5,6.8,2.5,6.5v-2C2.5,4.2,2.7,4,3,4h3.5C6.8,4,7,4.2,7,4.5v2C7,6.8,6.8,7,6.5,7z M3.5,6H6V5H3.5V6z"
        />
        <path
          d="M18.9,14H2.1C0.9,14,0,13.1,0,11.9l0,0V2.1C0,0.9,0.9,0,2.1,0h0h16.8C20.1,0,21,0.9,21,2.1c0,0,0,0,0,0v9.8 C21,13.1,20.1,14,18.9,14z M2.1,1C1.5,1,1,1.5,1,2.1l0,0v9.8C1,12.5,1.5,13,2.1,13h16.8c0.6,0,1.1-0.5,1.1-1.1l0,0V2.1 C20,1.5,19.5,1,18.9,1l0,0H2.1z"
        />
      </svg>
      Button with icon
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="secondary button  "
      onClick={[MockFunction]}
    >
      <svg
        viewBox="0 0 18.4 18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Share
        </title>
        <path
          d="M14.7,12c-1-0.1-1.9,0.3-2.5,1.1L5.8,8.7C5.9,8.4,6,8.1,6,7.8l0,0l6.8-3.3c0.5,0.8,1.4,1.4,2.3,1.5h0.2c0.7,0,1.4-0.3,2-0.7 c1.3-1.1,1.4-3,0.3-4.2c-1.1-1.3-3-1.4-4.2-0.3c-0.8,0.7-1.2,1.8-1,2.9L5.9,6.8c-0.5-1.6-2.1-2.5-3.7-2c-1.6,0.5-2.5,2.1-2,3.7 c0.4,1.2,1.4,2,2.7,2.1H3c0.9,0,1.7-0.4,2.3-1l6.4,4.4c-0.1,0.3-0.2,0.5-0.2,0.8c-0.1,1.7,1.1,3.1,2.8,3.2c0,0,0,0,0,0h0.2 c1.7,0,3-1.3,3-3c0-0.7-0.3-1.5-0.8-2C16.2,12.4,15.5,12.1,14.7,12z M15.4,1c1.1,0,2,0.9,2,2s-0.9,2-2,2s-2-0.9-2-2S14.3,1,15.4,1 C15.4,1,15.4,1,15.4,1z M3,9.6c-1.1,0-2-0.9-2-2s0.9-2,2-2s2,0.9,2,2S4.1,9.6,3,9.6z M14.5,17c-1.1,0-2-0.9-2-2s0.9-2,2-2s2,0.9,2,2 S15.6,17,14.5,17z"
        />
      </svg>
      Button with icon
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="primary button  "
      disabled={true}
      onClick={[MockFunction]}
    >
      <svg
        viewBox="0 0 12 12.7"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Transfer
        </title>
        <path
          d="M0,2.9c0-0.3,0.2-0.5,0.5-0.5h9.8L8.6,0.9c-0.2-0.2-0.2-0.5,0-0.7c0,0,0,0,0,0c0.2-0.2,0.5-0.2,0.7,0c0,0,0,0,0,0l2.5,2.4 c0.2,0.2,0.2,0.5,0,0.7L9.3,5.7c-0.2,0.2-0.5,0.2-0.7,0c0,0,0,0,0,0C8.4,5.5,8.4,5.2,8.6,5c0,0,0,0,0,0l1.6-1.6H0.5 C0.2,3.4,0,3.1,0,2.9z"
        />
        <path
          d="M12,9.9c0,0.3-0.2,0.5-0.5,0.5H1.7l1.6,1.5c0.2,0.2,0.2,0.5,0,0.7c0,0,0,0,0,0c-0.2,0.2-0.5,0.2-0.7,0c0,0,0,0,0,0l-2.5-2.3 C0,10,0,9.7,0.2,9.5l2.5-2.4c0.2-0.2,0.5-0.2,0.7,0c0,0,0,0,0,0c0.2,0.2,0.2,0.5,0,0.7c0,0,0,0,0,0L1.7,9.4h9.8 C11.8,9.4,12,9.6,12,9.9z"
        />
      </svg>
      Button with icon
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="primary button small "
      onClick={[MockFunction]}
    >
      <svg
        viewBox="0 0 12 12.7"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Transfer
        </title>
        <path
          d="M0,2.9c0-0.3,0.2-0.5,0.5-0.5h9.8L8.6,0.9c-0.2-0.2-0.2-0.5,0-0.7c0,0,0,0,0,0c0.2-0.2,0.5-0.2,0.7,0c0,0,0,0,0,0l2.5,2.4 c0.2,0.2,0.2,0.5,0,0.7L9.3,5.7c-0.2,0.2-0.5,0.2-0.7,0c0,0,0,0,0,0C8.4,5.5,8.4,5.2,8.6,5c0,0,0,0,0,0l1.6-1.6H0.5 C0.2,3.4,0,3.1,0,2.9z"
        />
        <path
          d="M12,9.9c0,0.3-0.2,0.5-0.5,0.5H1.7l1.6,1.5c0.2,0.2,0.2,0.5,0,0.7c0,0,0,0,0,0c-0.2,0.2-0.5,0.2-0.7,0c0,0,0,0,0,0l-2.5-2.3 C0,10,0,9.7,0.2,9.5l2.5-2.4c0.2-0.2,0.5-0.2,0.7,0c0,0,0,0,0,0c0.2,0.2,0.2,0.5,0,0.7c0,0,0,0,0,0L1.7,9.4h9.8 C11.8,9.4,12,9.6,12,9.9z"
        />
      </svg>
      Button with icon
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="primary button  "
      onClick={[MockFunction]}
    >
      CTA CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="secondary button  "
      onClick={[MockFunction]}
    >
      CTA CTA character limit 20-25
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="quickLink"
      onClick={[MockFunction]}
    >
      <div>
        <svg
          viewBox="0 0 21 14"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>
            Card
          </title>
          <path
            d="M6.5,11H3c-0.3,0-0.5-0.2-0.5-0.5S2.7,10,3,10h3.5C6.8,10,7,10.2,7,10.5S6.8,11,6.5,11z"
          />
          <path
            d="M17.5,11h-7.1c-0.3,0-0.5-0.2-0.5-0.5s0.2-0.5,0.5-0.5h7.1c0.3,0,0.5,0.2,0.5,0.5S17.8,11,17.5,11z"
          />
          <path
            d="M6.5,7H3C2.7,7,2.5,6.8,2.5,6.5v-2C2.5,4.2,2.7,4,3,4h3.5C6.8,4,7,4.2,7,4.5v2C7,6.8,6.8,7,6.5,7z M3.5,6H6V5H3.5V6z"
          />
          <path
            d="M18.9,14H2.1C0.9,14,0,13.1,0,11.9l0,0V2.1C0,0.9,0.9,0,2.1,0h0h16.8C20.1,0,21,0.9,21,2.1c0,0,0,0,0,0v9.8 C21,13.1,20.1,14,18.9,14z M2.1,1C1.5,1,1,1.5,1,2.1l0,0v9.8C1,12.5,1.5,13,2.1,13h16.8c0.6,0,1.1-0.5,1.1-1.1l0,0V2.1 C20,1.5,19.5,1,18.9,1l0,0H2.1z"
          />
        </svg>
      </div>
      Report a debit card lost or stolen
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="quickLink"
      onClick={[MockFunction]}
    >
      <div>
        <svg
          viewBox="0 0 12 12.7"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>
            Transfer
          </title>
          <path
            d="M0,2.9c0-0.3,0.2-0.5,0.5-0.5h9.8L8.6,0.9c-0.2-0.2-0.2-0.5,0-0.7c0,0,0,0,0,0c0.2-0.2,0.5-0.2,0.7,0c0,0,0,0,0,0l2.5,2.4 c0.2,0.2,0.2,0.5,0,0.7L9.3,5.7c-0.2,0.2-0.5,0.2-0.7,0c0,0,0,0,0,0C8.4,5.5,8.4,5.2,8.6,5c0,0,0,0,0,0l1.6-1.6H0.5 C0.2,3.4,0,3.1,0,2.9z"
          />
          <path
            d="M12,9.9c0,0.3-0.2,0.5-0.5,0.5H1.7l1.6,1.5c0.2,0.2,0.2,0.5,0,0.7c0,0,0,0,0,0c-0.2,0.2-0.5,0.2-0.7,0c0,0,0,0,0,0l-2.5-2.3 C0,10,0,9.7,0.2,9.5l2.5-2.4c0.2-0.2,0.5-0.2,0.7,0c0,0,0,0,0,0c0.2,0.2,0.2,0.5,0,0.7c0,0,0,0,0,0L1.7,9.4h9.8 C11.8,9.4,12,9.6,12,9.9z"
          />
        </svg>
      </div>
      Make a transfer
    </button>
  </div>,
  <div
    className="row"
  >
    <button
      className="quickLink"
      onClick={[MockFunction]}
    >
      <div>
        <svg
          viewBox="0 0 18.4 18"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>
            Share
          </title>
          <path
            d="M14.7,12c-1-0.1-1.9,0.3-2.5,1.1L5.8,8.7C5.9,8.4,6,8.1,6,7.8l0,0l6.8-3.3c0.5,0.8,1.4,1.4,2.3,1.5h0.2c0.7,0,1.4-0.3,2-0.7 c1.3-1.1,1.4-3,0.3-4.2c-1.1-1.3-3-1.4-4.2-0.3c-0.8,0.7-1.2,1.8-1,2.9L5.9,6.8c-0.5-1.6-2.1-2.5-3.7-2c-1.6,0.5-2.5,2.1-2,3.7 c0.4,1.2,1.4,2,2.7,2.1H3c0.9,0,1.7-0.4,2.3-1l6.4,4.4c-0.1,0.3-0.2,0.5-0.2,0.8c-0.1,1.7,1.1,3.1,2.8,3.2c0,0,0,0,0,0h0.2 c1.7,0,3-1.3,3-3c0-0.7-0.3-1.5-0.8-2C16.2,12.4,15.5,12.1,14.7,12z M15.4,1c1.1,0,2,0.9,2,2s-0.9,2-2,2s-2-0.9-2-2S14.3,1,15.4,1 C15.4,1,15.4,1,15.4,1z M3,9.6c-1.1,0-2-0.9-2-2s0.9-2,2-2s2,0.9,2,2S4.1,9.6,3,9.6z M14.5,17c-1.1,0-2-0.9-2-2s0.9-2,2-2s2,0.9,2,2 S15.6,17,14.5,17z"
          />
        </svg>
      </div>
      Share
    </button>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="content"
        >
          <div
            className="left"
          >
            <img
              alt="Tile graphic"
              src="graphic_1.png"
            />
          </div>
          <div>
            <h1
              className="lead"
            >
              See our tutorial
            </h1>
            <div>
              Step by step illustrations on how to use Online Banking
            </div>
            <button
              className="secondary button  linkOnDesktop"
              onClick={[MockFunction]}
            >
              View demo
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="content"
        >
          <div
            className="left"
          >
            <img
              alt="Tile graphic"
              src="graphic_2.png"
            />
          </div>
          <div>
            <h1
              className="lead"
            >
              Speed up login
            </h1>
            <div>
              Save your details on the login screen and you'll only need your PINsentry device and card to login
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="content"
        >
          <div
            className="left"
          >
            <img
              alt="Tile graphic"
              src="graphic_2.png"
            />
          </div>
          <div>
            <h6
              className="lead"
            >
              Speed up login
            </h6>
            <div>
              Save your details on the login screen and you'll only need your PINsentry device and card to login
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="content"
        >
          <img
            alt="Tile graphic"
            src="graphic_3.png"
          />
          <h1>
            <button
              className="lead lightHover"
              onClick={[MockFunction]}
            >
              Overdraft
            </button>
          </h1>
          Get an overdraft of up to 
          <b>
            £2,000
          </b>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="content"
        >
          <img
            alt="Tile graphic"
            src="graphic_3.png"
          />
          <h6>
            <button
              className="lead lightHover"
              onClick={[MockFunction]}
            >
              Overdraft
            </button>
          </h6>
          Get an overdraft of up to 
          <b>
            £2,000
          </b>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="cover"
        >
          <img
            alt="Tile graphic"
            src="promo_image.png"
          />
        </div>
        <div
          className="content"
        >
          <h1>
            <button
              className="lead "
              onClick={[MockFunction]}
            >
              1st card title goes here
            </button>
          </h1>
          <div>
            Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse nec tortor urna. Ut laoreet sodales nisi, nex quis iaculi. Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse romani.
          </div>
          <button
            className="primary button  "
            onClick={[MockFunction]}
          >
            Call to action
          </button>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="cover"
        >
          <img
            alt="Tile graphic"
            src="promo_image.png"
          />
        </div>
        <div
          className="content"
        >
          <h1>
            <button
              className="lead "
              onClick={[MockFunction]}
            >
              1st card title goes here
            </button>
          </h1>
          <div>
            Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse nec tortor urna. Ut laoreet sodales nisi, nex quis iaculi. Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse romani.
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="cover"
        >
          <img
            alt="Tile graphic"
            src="promo_image.png"
          />
        </div>
        <div
          className="content"
        >
          <h6>
            <button
              className="lead "
              onClick={[MockFunction]}
            >
              1st card title goes here
            </button>
          </h6>
          <div>
            Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse nec tortor urna. Ut laoreet sodales nisi, nex quis iaculi. Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse romani.
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="cover"
        >
          <img
            alt="Tile graphic"
            src="promo_image.png"
          />
          <div
            className="badge"
          >
            <img
              alt="Badge"
              src="logo.png"
            />
          </div>
        </div>
        <div
          className="content"
        >
          <h1>
            <button
              className="lead lightHover"
              onClick={[MockFunction]}
            >
              Cashback
            </button>
          </h1>
          <div>
            Get cashback when shopping at participant retailers
          </div>
          <button
            className="primary button  "
            onClick={[MockFunction]}
          >
            Call to action
          </button>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="cover"
        >
          <img
            alt="Tile graphic"
            src="promo_image.png"
          />
          <div
            className="badge"
          >
            <img
              alt="Badge"
              src="logo.png"
            />
          </div>
        </div>
        <div
          className="content"
        >
          <h1>
            <button
              className="lead lightHover"
              onClick={[MockFunction]}
            >
              Cashback
            </button>
          </h1>
          <div>
            Get cashback when shopping at participant retailers
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="tile"
      >
        <div
          className="cover"
        >
          <img
            alt="Tile graphic"
            src="promo_image.png"
          />
          <div
            className="badge"
          >
            <img
              alt="Badge"
              src="logo.png"
            />
          </div>
        </div>
        <div
          className="content"
        >
          <h6>
            <button
              className="lead lightHover"
              onClick={[MockFunction]}
            >
              Cashback
            </button>
          </h6>
          <div>
            Get cashback when shopping at participant retailers
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
